_reports/RJ722.md

```diff
@@ -0,0 +1,388 @@
+   - details: Pre-format logs
+
+ - 48:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/e50a54e82f7d63ec2892d5fa28f49ab09d277708
+   - details: Update localstack_analysis.md
+
+ - 49:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/b47fc3691055d618576f9142d36832cd1f94fd8c
+   - details: Make report conform to the given template
+
+ - 50:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/513e41cc4904760196e591602e3356bd1a504a5c
+   - details: Update localstack_analysis.md
+
+ - 51:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/3ab0a800db0c2200f0c666d4e198c8558c5fc325
+   - details: create analysis report for system design primer
+
+ - 52:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/e40be61b4e6e9cf97d689a328e8aa27c2637a85c
+   - details: Organise reports into folders.
+
+ - 53:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/85de1d41b14544e634a65fe5cb215dfee31eb2cf
+   - details: add header for localstack
+
+ - 54:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/89bfb08fc7fdcada02389bd4251632140a7da86c
+   - details: Add report for relational-networks
+
+ - 55:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/00a0492071c449ee34e36fec7a59254133be2f1b
+   - details: Add footer
+
+ - 56:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/43253a9384cc655c9e854766714cc3f5240e55bb
+   - details: "Add report for qutebrowser (#4)"
+
+ - 57:
+   - repo: vulture-analysis
+   - link: https://github.com/jendrikseipp/vulture-analysis/commit/5bfabaca9902017815cc01d3b38e92a21107d86d
+   - details: "Add report for Sharppy (#5)"
+---
+
+### Improving whitelisting infrastructure for Vulture
+
+#### Work Done
+
+<br>
+
+1.) Created a command line tool which takes in sip files as input and spits a
+whitelist with all the virtual functions defined in the [sip][sip] files. This
+tool was further used to create a whitelist for PyQt which can hereby be used
+with any project using PyQt. The tool is now completely tested and documented.
+
+<br>
+
+2.) Created `--make-whitelist` flag to enable reporting of the results in a
+format which can be directly used as a whitelist. This significantly eased the
+process of creating and maintaing a whitelist for a project.
+
+<br>
+
+3.) Implemented a ignoring mechanism for Vulture (`--ignore-names` and
+`--ignore-decorators`) - thereby enabling users to omit the result(s) which
+match with the given argument(s) (a glob pattern).
+
+<br>
+
+4.) Increased test coverage of Vulture from already excellent 95% to 100. Also,
```

Comment:
Minor: 100"%"

To be a bit pedantic: The python3.7 one is not fix-tests (as they were never tested before) but "ensure support for py3.7 in CI/CD" or similar
And should probably be merged with the below point